cob_light/ros/src/cob_light.cpp

```diff
@@ -191,12 +195,12 @@ class LightControl
     //Subscribe to LightController Command Topic
     _sub = _nh.subscribe("command", 1, &LightControl::topicCallback, this);
 
-    //Subscribe to LightController Command Topic
-    _sub_mode = _nh.subscribe("light", 1, &LightControl::topicCallbackMode, this);
-
     //Advertise light mode Service
     _srvServer = _nh.advertiseService("set_light", &LightControl::serviceCallback, this);
 
+    //Advertise stop mode service
+    _srvStopMode = _nh.advertiseService("stop_mode", &LightControl::stopMode, this);
```

Comment:
nope, because the action will return succeeded as soon as the requested mode started